app/controllers/screenings_controller.rb

```diff
@@ -1,4 +1,10 @@
 # frozen_string_literal: true
 
 class ScreeningsController < ApplicationController
+  include JSONAPI::Fetching
+
+  def index 
```

Comment:
```suggestion
  def index
```

Did you run rubocop on your project?